src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Services/SectionReviewOrchestrator.cs

```diff
@@ -0,0 +1,166 @@
+﻿using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using SFA.DAS.RoatpAssessor.Web.ApplyTypes;
+using SFA.DAS.RoatpAssessor.Web.ApplyTypes.Apply;
+using SFA.DAS.RoatpAssessor.Web.ApplyTypes.Enums;
+using SFA.DAS.RoatpAssessor.Web.Infrastructure.ApiClients;
+using SFA.DAS.RoatpAssessor.Web.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.RoatpAssessor.Web.Services
+{
+    public class SectionReviewOrchestrator : ISectionReviewOrchestrator
+    {
+        private readonly ILogger<SectionReviewOrchestrator> _logger;
+        private readonly IRoatpApplicationApiClient _applyApiClient;
+        private readonly ISupplementaryInformationService _supplementaryInformationService;
+
+        public SectionReviewOrchestrator(ILogger<SectionReviewOrchestrator> logger, IRoatpApplicationApiClient applyApiClient, ISupplementaryInformationService supplementaryInformationService)
+        {
+            _logger = logger;
+            _applyApiClient = applyApiClient;
+            _supplementaryInformationService = supplementaryInformationService;
+        }
+
+        public async Task<ReviewAnswersViewModel> GetReviewAnswersViewModel(GetReviewAnswersRequest request)
```

Comment:
Can this be split up into smaller functions please?

Reply:
I've split getting tabular data into a function. However on reviewing this block of code have raised several questions myself

Reply:
This orchestrator looks unwieldy to me.  The mapping stuff could be put into a method to help see what the real logic is doing maybe?

Reply:
In order to get current functionality tested, a ticket will be raised to address the finessing of this orchestrator.